ownCloudAppShared/Client/User Interface/ClientItemCell.swift

```diff
@@ -36,7 +37,8 @@ open class ClientItemCell: ThemeTableViewCell, ItemContainer {
 	private let smallSpacing : CGFloat = 2
 	private let iconViewWidth : CGFloat = 40
 	private let detailIconViewHeight : CGFloat = 15
-	private let moreButtonWidth : CGFloat = 60
+	private let moreButtonWidth : CGFloat = 45
```

Comment:
I figured out, that changing this button size, will lead in the right aligned button will no longer aligned in a row.
I would change this back and set 
```
private let moreButtonWidth : CGFloat = 60


moreButtonWidthConstraint?.constant = showRevealButton ? revealButtonWidth : moreButtonWidth
```

I can commit this change, because I already tested it.

Reply:
I just merged your suggestion branch. Thanks!